src/core/Parameters.cc

```diff
@@ -67,6 +67,30 @@ const Value& Parameters::get(const std::string& name) const
   return *value;
 }
 
+const Value& Parameters::get(const std::initializer_list<std::string> names) const
+{
+  std::string match;
+  for (std::string name: names) {
+    boost::optional<const Value&> value = lookup(name);
+    if (value && value->isDefined()) {
+      if (match.empty()) {
+        match = name;
+      } else {
+        LOG(message_group::Error, loc, documentRoot(),
+            "Specified both %1$s and %2$s", quoteVar(match), quoteVar(name));
+      }
+    }
+  }
+
+  for (std::string name : names) {
+    boost::optional<const Value&> value = lookup(name);
+    if (value && value->isDefined()) {
+      return *value;
+    }
+  }
```

Comment:
Instead of iterating and looking up twice, can't you just keep the boost::optional around from earlier and return the value stored in that?

Reply:
That does seem desirable, but my C++ is weak enough that I don't understand the rules for keeping around Value& values.  I'll try something and see if the compiler lets me.  (But I think I already tried that, and failed.)

Reply:
I would try smth. like this (not tested):
```
  std::string match_name;
  boost::optional<const Value&> match_value;
  for (const std::string& name : names) {
    boost::optional<const Value&> value = lookup(name);
    if (value && value->isDefined()) {
      if (!match_value) {
        match_name = name;
        match_value = value;
      } else {
        LOG(message_group::Error, loc, documentRoot(),
            "Specified both %1$s and %2$s", quoteVar(match_name), quoteVar(name));
      }
    }
  }

  return match_value ? *match_value : Value::undefined;
```

Reply:
..and we should probably port from boost::optional to std::optional at some point

Reply:
Wind and fire mean that I haven't had power since Tuesday evening. Not in danger, but may be offline for another couple of days.

Reply:
Holy crap - you live in that part of CA. Stay safe, nobody is loosing sleep over late PRs :)

Reply:
Done.  (Not the std::optional part, because that requires more extensive changes.)